start

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cd client
+npm i
```

Comment:
Why do you need to npm i? Docker does that for you in containers!!

Reply:
You don't *have* to `npm i` to get the app running, but if people are running it locally they probably want to mess with it, in which case they will probs want the packages installed on the host machine, i.e. outside of the containers